www/config.rb

```diff
@@ -224,3 +224,4 @@ def render_markdown(text)
 redirect 'demo/packaging-system/index.html', to: '/demo/packaging-system/steps/1'
 redirect 'demo/build-system/index.html', to: '/demo/build-system/steps/1'
 redirect 'demo/process-supervisor/index.html', to: '/demo/process-supervisor/steps/1'
+redirect 'legal', to: 'legal/licensing/'
```

Comment:
I think this has to be `legal/index.html`, but acceptance should tell us for sure. :smile:

Reply:
Yeah looks like that does need to be there:

http://www.acceptance.habitat.sh/legal

Changing `'legal'` to `'legal/index.html'` does the trick, verified locally.